### ISSUE TITLE
Disabling Autosave not working

### ISSUE THREAD

**Issue by treetop1500** [froala/KMSFroalaEditorBundle#26]
Setting the config parameter for 'saveInterval' to 0 (zero) is not disabling autosave as it should according to the Froala Editor docs.

https://www.froala.com/wysiwyg-editor/docs/options#saveInterval

I noticed this after implementing the image and file upload concepts, that invoking those tools was causing the autosave to attempt to occur, then resulting in the alert message "The saveURL is not defined."  I do not have the saveURL defined because I don't want to perform autosave.

**Comment by samarhxc** [comment]
Fixed.

**Comment by treetop1500** [comment]
This has recently started occurring again.  I had the dev-master branch running on a site, and we started receiving an error stating "Error 2: Something went wrong during save". Tracked it down to auto save trying to run (even tho "saveInterval" is set to 0).  I changed it to version ~2.5 and updated, made sure saveInterval was set to 0 in the config and commented out saveUrl and saveMethod, and now I'm receiving the error "Error 1. Missing saveUrl option".  So the auto save is attempting to run again.

Thank you.
